assets/src/components/streetViewButton.tsx

```diff
@@ -16,16 +16,16 @@ export interface StreetViewButtonProps extends GeographicCoordinateBearing {
   className?: string
   text?: string
   title?: string
-  children?: ReactElement
+  children?: ReactNode
 }
 
-const StreetViewButton = ({
+export const StreetViewButton = ({
   className,
   text,
   children,
   title,
   ...worldPosition
-}: StreetViewButtonProps): ReactElement<HTMLElement> => {
+}: StreetViewButtonProps) => {
```

Comment:
**question:** is it possible to keep an explicit return type? Not that we've been super consistent about what we use for return types on components.

Reply:
I encountered a storybook github issue where one of the solutions was to remove the return type from the function component, I'm not sure it was actually needed here any more, but I think it's better to leave the return type unspecified in the case of react.

I don't think this is the right return type to use either, so I'd be open to changing it given storybook still works.

Reply:
That's fair - I'll confess that I don't really know which of the many `React` and `JSX` types is strictly correct for different use cases.